packages/atomic/src/components/formats/atomic-format-unit.tsx

```diff
@@ -0,0 +1,60 @@
+import {Component, Element, Prop, State, h} from '@stencil/core';
+import {dispatchNumberFormatEvent} from './format-common';
+
+/**
+ * The `atomic-format-unit` component is used for formatting numbers with units.
+ * It will set the numerical format on compatible parents according to the options.
```

Comment:
The numerical format of compatible parents will be set according to the properties of this component.